### PR TITLE
Karma: Add Copy and Paste tests

### LINKED ISSUE #5564: Karma: Canvas Copy and Paste Tests
Create karma tests for Copy and Paste. Should do a base test on copy and pasting elements without animations. Should also test copy and pasting an element with every FG & BG animation.

### PR DESCRIPTION
**Pull request by maxyinger**
## Summary
Adds karma tests for copy and paste actions as well as duplicate/clone actions. These both have the same external functionality to the user, but have different internal implementations. These tests verify that both behave the exact same to the user.

This also tests these actions with all foreground and background animations visible in the effect chooser. This is nice cus as we add or remove effects from the effects panel, this test ensures us that none of the outward facing animations break the app on copy & paste or duplicate.

These new tests already pointed out some dead code that wasn't being used for background zoom, so that's nice!

## Relevant Technical Choices
Just followed exiting patterns.

## To-do
Page duplication is similar to copy & paste and duplicate/clone actions, but has a different underlying implementation. 

Here's the ticket to add karma tests for it: https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/5737

Just want to point out here, that these tests don't cover page duplication although the functionality is super similar.

## User-facing changes
None

## Testing Instructions
None - covered in CI karma tests

---

Fixes #5564

### CODE REVIEW

**Review by BrittanyIRL — APPROVED** [review]
These are awesome adds! 

Heads up that there's a misspelling of `copied` as `coppied`

File: assets/src/edit-story/karma/copyAndPaste.cuj.karma.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+ * @param {(el: HTMLElement, i: number) => Promise<void>} op - async operation
+ * @return {Promise<void>}
+ */
+function sequencedForEach(htmlCollection, op) {
+  return new Promise((res) => {
+    Array.from(htmlCollection)
+      .map((htmlElement, i) => async () => {
+        await op(htmlElement, i);
+      })
+      .reduce((p, fn) => p.then(fn), Promise.resolve())
```
Comment (carloskelly13) [inline assets/src/edit-story/karma/copyAndPaste.cuj.karma.js:43]: This right here is a crash course in promises and async. But super cool.

**Review by carloskelly13 — APPROVED** [review]

**Review by carloskelly13 — APPROVED** [review]
This is great and thank you for commenting each step in these tests. It's a pretty complex flow and this will help make it clearer for readers down the line.

**Comment by maxyinger** [comment]
> These are awesome adds!
> 
> Heads up that there's a misspelling of `copied` as `coppied`

lol thanks. I can't spell for the life of me

File: assets/src/edit-story/karma/copyAndPaste.cuj.karma.js (hunk 1 above)
Comment (maxyinger) [reply assets/src/edit-story/karma/copyAndPaste.cuj.karma.js:43]: oh yea. took me a sec to figure out. might be nice to put a little comment of the transformation on there:
```
// [p1, p2, p3, p4] -> promise.resolve().then(p1).then(p2).then(p3).then(p4)
```
